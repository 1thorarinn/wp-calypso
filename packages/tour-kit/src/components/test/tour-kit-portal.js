/**
 * @jest-environment jsdom
 */
import { shallow } from 'enzyme';
import TourKitFrame from '../tour-kit-frame';
import TourKitPortal from '../tour-kit-portal';

describe( 'TourKitPortal', () => {
	test( 'should render', () => {
		const wrapper = shallow( <TourKitPortal /> );
		expect( wrapper ).toMatchSnapshot();
		expect( wrapper.type() ).toEqual( 'div' );
	} );

	test( 'should have TourKitFrame with config', () => {
		const testConfig = {
			foo: 'bar',
		};

		const wrapper = shallow( <TourKitPortal config={ testConfig } /> );
		expect( wrapper.find( TourKitFrame ) ).toHaveLength( 1 );
		expect( wrapper.find( TourKitFrame ).prop( 'config' ) ).toEqual( testConfig );
	} );
} );
