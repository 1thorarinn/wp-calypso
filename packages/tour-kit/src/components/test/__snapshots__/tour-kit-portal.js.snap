// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TourKitPortal should render 1`] = `
<div>
  <Portal
    containerInfo={<div />}
  >
    <TourKitFrame />
  </Portal>
</div>
`;
